### ISSUE TITLE
iotop complains "CONFIG_TASK_DELAY_ACCT not enabled in kernel", but it is

### ISSUE THREAD

**Issue by Mek101** [void-linux/void-packages#34663]
### System
* xuname: Void 5.15.9_1 x86_64 AuthenticAMD uptodate rFF
* package: iotop-0.6_9

### Expected behavior
`iotop` should report the IO percentage

### Actual behavior
`iotop` complains `CONFIG_TASK_DELAY_ACCT not enabled in kernel, cannot determine SWAPIN and IO %`, but according to the kernel config, it enabled
```
$ zgrep CONFIG_TASK_DELAY_ACCT /proc/config.gz
CONFIG_TASK_DELAY_ACCT=y
```

### Steps to reproduce the behavior
Install the `iotop` package and run it with root privileges

**Comment by ericonr** [comment]
```
class Stats(DumpableObject):
    members_offsets = [
        ('blkio_delay_total', 40),
        ('swapin_delay_total', 56),
        ('read_bytes', 248),
        ('write_bytes', 256),
        ('cancelled_write_bytes', 264)
    ]

    has_blkio_delay_total = False

    def __init__(self, task_stats_buffer):
        sd = self.__dict__
        for name, offset in Stats.members_offsets:
            data = task_stats_buffer[offset:offset + 8]
            sd[name] = struct.unpack('Q', data)[0]

        # This is a heuristic to detect if CONFIG_TASK_DELAY_ACCT is enabled in
        # the kernel.
        if not Stats.has_blkio_delay_total:
            Stats.has_blkio_delay_total = self.blkio_delay_total != 0
```

It's poking at a lot of things, it's not surprising that the heuristic broke.

https://repo.or.cz/w/iotop.git has some more commits, do you think you could test if the more recent version works fine?

**Comment by Mek101** [comment]
It's worse: `CONFIG_TASK_DELAY_ACCT and kernel.task_delayacct sysctl not enabled in kernel, cannot determine SWAPIN and IO %`

**Comment by ericonr** [comment]
Not sure how to go about fixing this :/

**Comment by Mek101** [comment]
They had the issue and solved it: https://github.com/Tomas-M/iotop/issues/21

Apparently `iotop` needs `kernel.task_delayacct` to be enabled, however since kernel 5.15 it seems to be disabled by default

**Comment by Mek101** [comment]
Add `delayacct` to the kernel command line as a workaround

**Comment by bbonev** [comment]
> They had the issue and solved it: [Tomas-M/iotop#21](https://github.com/Tomas-M/iotop/issues/21)
> 
> Apparently `iotop` needs `kernel.task_delayacct` to be enabled, however since kernel 5.15 it seems to be disabled https://www.kernel.org/doc/html/v5.15/accounting/delay-accounting.html

The above link points to the C rewrite of `iotop` (packaged as `iotop-c` in many Linux distributions but not in Void Linux) while the python one is located at https://repo.or.cz/iotop.git; the OP obviously talks about the python version of `iotop`.
The fix mentioned above is for the C version where the user can toggle the `task_delayacct` sysctl with a shortcut. The python version only gives a warning in case the sysctl is off and SWAPIN/IO do not work (the fix is not in a released version yet, one option is to package the HEAD of the python iotop's repo).

Hope I could reduce the confusion...
